examples/flutter_gallery/lib/demo/cupertino/cupertino_navigation_demo.dart

```diff
@@ -131,17 +131,21 @@ class CupertinoDemoTab1 extends StatelessWidget {
             largeTitle: const Text('Colors'),
             trailing: const ExitButton(),
           ),
-          new SliverList(
-            delegate: new SliverChildBuilderDelegate(
-              (BuildContext context, int index) {
-                return new Tab1RowItem(
-                  index: index,
-                  lastItem: index == 49,
-                  color: colorItems[index],
-                  colorName: colorNameItems[index],
-                );
-              },
-              childCount: 50,
+          new SliverPadding(
+            // Top media query padding already consumed by CupertinoSliverNavigationBar.
+            padding: MediaQuery.of(context).removePadding(removeTop: true).padding,
```

Comment:
this doesn't seem right... we're consuming the padding, but we're not actually updating the MediaQuery for the children.

Reply:
oh, i see, this is the demo file, not a widget. probably ok then.